rest-api/src/main/java/gov/cms/qpp/conversion/api/model/Metadata.java

```diff
@@ -393,6 +396,33 @@ public void setCpcProcessed(Boolean cpcProcessed) {
 		this.cpcProcessed = cpcProcessed;
 	}
 
+	@DoNotEncrypt
+	@DynamoDBAttribute(attributeName = "CpcProcessed_CreateDate")
+	public String getCpcProcessedCreateDate() {
+		String combination = null;
+
+		if (cpcProcessed != null) {
+			combination = cpcProcessed.toString() + "#" + DateTimeFormatter.ISO_INSTANT.format(createdDate.toInstant());
+		}
+
+		return combination;
+	}
+
+	public void setCpcProcessedCreateDate(String combination) {
+
+		String[] split = combination.split("#");
+
+		if (split.length < 2) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 2 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)